internal/controller/controllers/agentserviceconfig_controller.go

```diff
@@ -239,6 +241,45 @@ func (r *AgentServiceConfigReconciler) Reconcile(origCtx context.Context, req ct
 		Reason:  aiv1beta1.ReasonReconcileSucceeded,
 		Message: msg,
 	})
+
+	for _, component := range []struct {
+		name          string
+		conditionType string
+		reason        string
+		fn            ComponentStatusFn
+	}{
+		{"agentinstalladmission", "Available", "DeploymentsUnhealthy", r.deploymentStatus},
+		{"agentinstalladmission", "Progressing", "DeploymentsUnhealthy", r.deploymentStatus},
+		{"assisted-image-service", "Available", "DeploymentsUnhealthy", r.deploymentStatus},
+		{"assisted-image-service", "Progressing", "DeploymentsUnhealthy", r.deploymentStatus},
```

Comment:
Will this label the deployment as unhealthy if the deployment is not progressing?
Does `Progressing` go to false when the deployment is successfully rolled out?

Reply:
IIRC, `Progressing` still remains `True`. My dev-scripts env is not working so can't validate it.

Reply:
Yup, that looks to be the case.
I just checked on a deployment in the cloud.

You don't need a full dev-scripts setup to run `oc get deploy` :wink: